{
  "exit_code": 1,
  "stdout": "",
  "stderr": "[src/tests/stmt/fn_name_conflict.moc:2:4] Error: Function name conflict. `Foo` defined elsewhere\n"
}